Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1751,7 +1781,16 @@ static void addValueForAnimationPropertyToList(const RenderStyle& style, CSSValu
                 list.append(valueForAnimationTimingFunction(*animation->timingFunction()));
         } else
             list.append(valueForAnimationTimingFunction(CubicBezierTimingFunction::defaultTimingFunction()));
-    } else
+    } else if (property == CSSPropertyAnimationRangeStart) {
```

Comment:
Can we change this to a switch statement?

Reply:
Fixed.